modules/statistics/RTPStatsCollector.js

```diff
@@ -242,6 +250,7 @@ export default function StatsCollector(
      * @private
      */
     this._getStatValue = this._defineGetStatValueMethod(keys);
+    this._getNewStatValue = this._defineNewGetStatValueMethod(keys);
```

Comment:
Did you find it more clear to have separate instance variables for the get stats method instead of sharing one but conditionally selecting which _defineXXX got called?

Reply:
Yep, makes sense. I was worried about spreading the if statements deciding which getStats implementation is used, but this thing can also be fixed. Will change that. Thanks.